video_v4l2.c

```diff
@@ -1355,6 +1355,23 @@ void v4l2_mutex_destroy(void) {
 #endif // HAVE_V4L2
 }
 
+static int v4l2_fps_set(struct context *cnt, struct video_dev *curdev) {
```

Comment:
This function would need to go above row 1339 in order to be part of the large `#if` for `HAVE_V4l2`